build.sbt

```diff
@@ -20,9 +20,9 @@ scalacOptions := Seq(
   "-deprecation"
 , "-unchecked"
 , "-Xfatal-warnings"
-, "-Xlint"
+, "-Xlint:_,-unused"
```

Comment:
We don't want unused errors because the exercises import useful things but don't use them (until the exercises are implemented)